src/app/main/time-groups/edit-time-group/edit-time-group.component.html

```diff
@@ -111,28 +114,30 @@ <h3 class="steel weight-bold size-default title-with-btn mb-4">{{'tr_assign_empl
           <div class="row">
             <div class="col-md-3">
               <div class="form-group">
-                <label>{{'tr_number' | translate}}</label>
+                <label>{{'tr_name' | translate}}</label>
                 <div class="input-with-icon bg-white width-100">
-                  <input class="text" (keyup)="filterEmployees($event,'by_number')" class="s-control"
-                    placeholder="{{'tr_search_by_number' | translate}}">
+                  <input class="text" type="text" class="s-control"  name ="employeeName" (input)="filterEmployees($event,'by_name')"
+                    placeholder="{{'tr_search_by_name' | translate}}">
                 </div>
               </div>
             </div>
             <div class="col-md-3">
               <div class="form-group">
-                <label>{{'tr_name' | translate}}</label>
+                <label>{{'tr_number' | translate}}</label>
                 <div class="input-with-icon bg-white width-100">
-                  <input class="text" class="s-control" (keyup)="filterEmployees($event,'by_name')"
-                    placeholder="{{'tr_search_by_name' | translate}}">
+                  <input class="text" type="text" (input)="filterEmployees($event,'by_number')" name ="employeeNumber" class="s-control"
+                    placeholder="{{'tr_search_by_number' | translate}}">
                 </div>
               </div>
             </div>
+         
+            
             <div class="col-md-3">
               <div class="form-group">
                 <label>{{'tr_department' | translate}}</label>
                 <div class="input-with-icon bg-white width-100">
                   <ng-select class="s-control s-select"
-                    (change)="filterEmployees(selectedDepartment,'by_department_id')" bindValue="id"
+                    (change)="filterEmployees(selectedDepartment,'by_department_id')" 
```

Comment:
You don't need `bindValue="id"` ?

Reply:
Bind value will only return the selected id as the selectedDepartment but I need the whole object to search by id (backend search) and by name (frontend search)

Reply:
I think you need to add bindLabel  `bindValue="id" (scrollToEnd)=nextBatch() bindLabel="name"`